openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -104,19 +104,60 @@ export const getLoggedInUserPermissions: Function =
 
 export const getInitialUsers: Function = (): Promise<AxiosResponse> => {
   return APIClient.get(
-    `/search/query?q=*&from=0&size=5&index=user_search_index`
+    `/search/query?q=*&from=0&size=5&index=${SearchIndex.USER}`
   );
 };
+export const getInitialEntity: Function = (): Promise<AxiosResponse> => {
+  return APIClient.get(
+    `/search/query?q=*&from=0&size=5&index=${SearchIndex.TABLE}`
+  );
+};
+
+export const getSuggestedUsers = (term: string): Promise<AxiosResponse> => {
+  return APIClient.get(`/search/suggest?q=${term}&index=${SearchIndex.USER}`);
```

Comment:
Actually we should remove returnType Function from everywhere, bcz otherwise typescript will not read actual returntype of response i.e. Promise<AxiosResponse>

Reply:
okay then can we please remove return type for `getInitialEntity`?